manager/assets/modext/widgets/media/modx.browser.js

```diff
@@ -58,6 +58,7 @@ MODx.browser.View = function(config) {
         }
         ,tpl: MODx.config.modx_browser_default_viewmode === 'list' ? this.templates.list : this.templates.thumb
         ,itemSelector: MODx.config.modx_browser_default_viewmode === 'list' ? 'div.modx-browser-list-item' : 'div.modx-browser-thumb-wrap'
+        ,loadingText : '<div style="padding:10px;"><h4>'+_('loading')+'</h4></div>'
```

Comment:
I'm just keeping it consistent with how the "Error/Empty" message loads. This line could be taken out because I don't think it gets triggered very often anyway.

Reply:
I've just removed it